lib/elixir/lib/code/normalizer.ex

```diff
@@ -182,6 +182,20 @@ defmodule Code.Normalizer do
     end
   end
 
+  # Tuples
+  defp do_normalize({:{}, meta, args} = quoted, state) do
+    {last_arg, args} = List.pop_at(args, -1)
+
+    with [{{:__block__, key_meta, _}, _} | _] <- last_arg, :keyword <- key_meta[:format] do
+      args = normalize_kw_args(args, state)
+      kw_list = normalize_kw_args(last_arg, state)
+      {:__block__, [], [{:{}, meta, args ++ kw_list}]}
+    else
+      _ ->
+        normalize_call(quoted, state)
```

Comment:
Actually, shouldn’t normalize call already handle keywords?

Reply:
It does handle them, but it won't process `:{}` forms the way we are doing it here due to `allow_keyword?` excluding it. We'd have to add an additional clause to the `cond` in line 314 with the same code we have here.